make_ghpages/mod/templates/base.html

```diff
@@ -38,9 +39,16 @@ <h1>
     {% block footer %}
     <footer class="footer">
     <hr>
-    This page is part of the <a href="{{ app_registry_url }}">BIG-MAP registry</a>, the official registry of <a href="{{ big_map_url }}">BIG-MAP</a> applications.
-    <br>
-    The BG-MAP project has received funding from the European Union’s <a href="https://ec.europa.eu/programmes/horizon2020/en" target="_blank">Horizon 2020 research and innovation programme</a> under grant agreement <a href="https://cordis.europa.eu/project/id/957189" target="_blank">No 957189</a>.
+    <div class="side-by-side">
+      <div>
+        <img src="{% block eulogoimagesrc %}static/img/eu-logo.png{% endblock %}" style="width: 60px; vertical-align: middle;" width="60px">
+      </div>
+    <div style="width: 80%;">
+      This page is part of the <a href="{{ app_registry_url }}">BIG-MAP registry</a>, the official registry of <a href="{{ big_map_url }}">BIG-MAP</a> applications.
+      <br>
+      The BIG-MAP project has received funding from the European Union’s <a href="https://ec.europa.eu/programmes/horizon2020/en" target="_blank">Horizon 2020 research and innovation programme</a> under grant agreement <a href="https://cordis.europa.eu/project/id/957189" target="_blank">No 957189</a>.
```

Comment:
I actually switched to the "This project ..." language in the README, because I was under the impression that this is the correct way of acknowledging this kind of project. Can you verify the correct language, please? We can make a separate PR to fix it in both the README and in the base template if you like. I believe either [Article 38.1.2](https://ec.europa.eu/research/participants/docs/h2020-funding-guide/grants/grant-management/acknowledge-funding_en.htm) is applicable.

Reply:
Unfortunately I think it *does* matter actually. This just came up again in a recent conversation with participants from a different project who reported that they had slightly adjusted the wording (exactly like here to include the project name) and the EC complained that the formulation was not _exactly_ as required. Mistakes in this matter are serious and can apparently even lead to withholding of funding.

Reply:
Ah ok - wow they are strict! Maybe good if you could mention this in the next AiiDA and MC meetings for others to know